content/actions/reference/workflow-syntax-for-github-actions.md

```diff
@@ -619,6 +619,27 @@ jobs:
         uses: docker://gcr.io/cloud-builders/gradle
 ```
 
+#### Example using action inside a different private repository as the workflow
+
+Your workflow must checkout the private repository and reference the action locally.
+
+{% raw %}
+```yaml
+jobs:
+  my_first_job:
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          repository: octocat/my-private-repo
+          ref: v1.0
+          token: ${{ secrets.GITHUB_PERSONAL_ACCESS_TOKEN }}
```

Comment:
optionally you can change this, since we encourage users to use GITHUB_TOKEN instead of a personal access token if possible:
```suggestion
          token: ${{ secrets.GITHUB_TOKEN }}
```